tests/core/test_config.py

```diff
@@ -110,3 +113,149 @@ def testDefaultChecks(self):
 
         for c in DEFAULT_CHECKS:
             self.assertTrue(c in init_checks_names)
+
+
+TMP_DIR = tempfile.gettempdir()
+TEMP_3RD_PARTY_CHECKS_DIR = '%s/dd-agent-tests/3rd-party' % TMP_DIR
```

Comment:
`os.path.join(TMP_DIR, 'dd-agent-tests', '3rd-party')` would be slightly better here

(same thing for the 3 lines below)